docs/src/main/paradox/scala/http/common/marshalling.md

```diff
@@ -1,9 +1,9 @@
 # Marshalling
 
-"Marshalling" is the process of converting a higher-level (object) structure into some kind of lower-level
-representation, often a "wire format". Other popular names for it are "Serialization" or "Pickling".
+Marshalling is the process of converting a higher-level (object) structure into some kind of lower-level
+representation, often a "wire format". Other popular names for marshalling are "serialization" or "pickling".
```

Comment:
I have to fight German bias too :-)
But I'd argue that according to [the rules of capitalization](https://en.wikipedia.org/wiki/Capitalization_in_English) those words should be lowercase.